src/js/tracks/text-track.js

```diff
@@ -19,7 +19,25 @@ import XHR from 'xhr';
  * @param {String} srcContent webVTT file contents
  * @param {Track} track track to addcues to
  */
-const parseCues = function(srcContent, track) {
+const parseCues = function(srcContent, track, retryNum) {
```

Comment:
actually, defaulting it to 3 (or 10) would make more sense plus get rid of some of the shenanigans I'm doing in lines 28-34.